app/components/UserAttendance/AttendanceStatus.js

```diff
@@ -8,27 +8,23 @@ import type { EventPool } from 'app/models';
 type AttendanceElementProps = {
   pool: EventPool,
   index: number,
-  toggleModal: number => void,
-  useModal: boolean,
-  isCount: boolean
+  toggleModal: number => void
 };
 
 const AttendanceElement = ({
-  // $FlowFixMe
-  pool: { name, registrations = 0, capacity = '∞' },
+  pool: { name, registrations, registrationCount, capacity },
   index,
-  toggleModal,
-  useModal,
-  isCount
+  toggleModal
 }: AttendanceElementProps) => {
-  const registrationCount =
-    isCount || !registrations ? registrations : registrations.length;
-  const Status = () => <strong>{`${registrationCount}/${capacity}`}</strong>;
+  const totalCount = registrations ? registrations.length : registrationCount;
```

Comment:
Is this necessary? Shouldn't these be the same number?

Reply:
Ish.

We can use `pool.registrationCount` always, but then we need to update it when websocket-actions with new registrations occurs (and i guess we should do that).

Reply:
https://github.com/webkom/lego-webapp/blob/master/app/reducers/registrations.js